appeals/api/src/server/openapi-types.ts

```diff
@@ -119,7 +119,7 @@ export interface ValidateDate {
 
 export interface AppellantCaseData {
 	casedata?: {
-		/** @example "ba35248c-0b50-4352-ac3e-89d5c8fd165f" */
+		/** @example "40df3b3d-fc9c-4e65-85db-4a9c383e0fcf" */
```

Comment:
Rare. Not that this isn't ok but why update this example? 😅

Reply:
It was part of the change when I regenerated the file, not sure why though :)